BeeTracker/app/src/main/res/layout/activity_home_page.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.blueprint.foe.beetracker.HomePageActivity">
+
+    <Button
+        android:id="@+id/button2"
```

Comment:
Can you give the button a more descriptive name? Maybe buttonSubmit for this one and buttonLearn for the next one?